Index: .gitignore
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.gitignore b/.gitignore
--- a/.gitignore	(revision df98770048867f58b6d8d6c07d01def0054b88b3)
+++ b/.gitignore	(date 1745350874030)
@@ -1,0 +1,15 @@
+# Ignore virtual environment directory
+.venv/
+
+# Ignore PyCharm project files
+.idea/
+
+# Byte-code files
+*.pyc
+__pycache__/
+
+# Ignore editor swap files
+*~
+
+# Ignore logs
+*.log
